scripts/img2img.py

```diff
@@ -69,7 +69,7 @@ def main():
     )
 
     parser.add_argument(
-        "--init-img",
+        "--init_img",
```

Comment:
This is a non-additive interface change.  I'd recommend against as it impacts anyone writing scripts to interact with the original code.  It's also beyond the intent of the commit.

Reply:
I don't think either of the submodule updates is needed, and the im watermark removal seems like it goes against the intent of ensuring the outputs are readily distinguishable as AI generated.  I'd at a minimum recommend suppressing it in the case it's not compatible, but as removed, it gets rid of the watermark even if someone has cuda working on their system.